wiki/Main_Page/id.md

```diff
@@ -1,46 +1,45 @@
 ---
-layout: main_page
-outdated: true
+layout: "main_page"
 ---
 
+<!-- Do not add any empty lines inside this div. -->
 <div class="wiki-main-page__blurb">
-osu! wiki adalah sebuah projek berdasarkan orang-orang yang berkontribusi dalam gim osu!. Tujuan akhirnya adalah menggantikan bagian FAQ yang lama, dan memungkinkannya menjadi sumber yang jauh lebih lengkap. Namun, untuk memungkinkan ini, kami membutuhkan bantuan dari semua orang yang mau berkontribusi!
-<br>
-Kamu dapat menggunakan akun osu! kamu untuk mulai berkontribusi!
+Welcome to the osu! wiki, a project containing a wide range of osu! related information.
+Selamat datang di osu! wiki, sebuah proyek yang berisi berbagai informasi terkait osu!
 </div>
 
 
 <div class="wiki-main-page__panels">
-<div class="wiki-main-page-panel wiki-main-page-panel--full">
+<div class="wiki-main-page-panel">
 
 
-# Mulai Permain
+# Memulai
```

Comment:
Feels better using "Pendahuluan" instead of Memulai. Incase "Pendahuluan" is more common for starting something.